### PR TITLE
:bug: Fix drop behavior

### LINKED ISSUE #46: [Bug] Throw & Drop
[Bug] Throw & Drop

### PR DESCRIPTION
**Pull request by SettingDust**
close #46

### CODE REVIEW

File: src/main/java/team/ebi/epicbanitem/check/Triggers.java [hunk 1]
```diff
@@ -46,7 +49,10 @@
     private static final SortedMap<String, CheckRuleTrigger> triggers = new TreeMap<>();
 
     @Inject
-    public Triggers(EventManager eventManager, PluginContainer pluginContainer) {
+    public Triggers(EventManager eventManager, PluginContainer pluginContainer, ThrowListener throwListener, DropListener dropListener) {
```
Comment (ginyai) [inline src/main/java/team/ebi/epicbanitem/check/Triggers.java]: For what?

Comment (SettingDust) [reply src/main/java/team/ebi/epicbanitem/check/Triggers.java]: Register listeners.
Split code to modules

Comment (ginyai) [reply src/main/java/team/ebi/epicbanitem/check/Triggers.java]: But should it be here?

Comment (SettingDust) [reply src/main/java/team/ebi/epicbanitem/check/Triggers.java]: Why not? Modular

**Pull request merged**